<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertBut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="removelineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getBut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>250</y>
      <width>271</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
